tinyagentos/decisions/decision_store.py

```
@@ -124,7 +130,7 @@ async def list(
         self,
         *,
         status: str | None = None,
-        project_id: str | None = None,
+        project_id: str | None | object = _UNSET,
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)